build.gradle

```diff
@@ -39,6 +39,9 @@ buildscript {
       okio: 'com.squareup.okio:okio:2.2.2',
       okio_1x: 'com.squareup.okio:okio:1.0.0',
       robolectric: 'org.robolectric:robolectric:4.0-alpha-3',
+      lint_api: 'com.android.tools.lint:lint-api:26.6.2',
+      lint: 'com.android.tools.lint:lint:26.6.2',
+      lint_test: 'com.android.tools.lint:lint-tests:26.6.2',
```

Comment:
It might be better to use array here:
```
lint: [
    api: 'com.android.tools.lint:lint-api:26.6.2',
    core: 'com.android.tools.lint:lint:26.6.2',
    test: 'com.android.tools.lint:lint-tests:26.6.2',
],

// Later in build.gradle:

compileOnly deps.lint.api

testImplementation deps.lint.core
testImplementation deps.lint.test
```